src/components/Layout/NavLink/index.jsx

```diff
@@ -8,7 +8,9 @@ import { inject } from 'mobx-react/index';
 import styles from './index.scss';
 
 @translate()
-@inject('rootStore')
+@inject(({ rootStore }) => ({
+  user: rootStore.user
+}))
 @observer
 export default class NavLink extends Component {
```

Comment:
`NavLink`, the name is same with `react-router-dom`'s NavLink

Reply:
ok.